x-pack/plugin/ccr/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/ccr/AutoFollowIT.java

```diff
@@ -163,12 +172,142 @@ public void testPutAutoFollowPatternThatOverridesRequiredLeaderSetting() throws
         );
     }
 
+    public void testDataStreams() throws Exception {
```

Comment:
Can you add a test where an auto-follow pattern is added after the leader cluster has created a data stream?